src/screens/crew6/relayScreen.cpp

```diff
@@ -218,15 +218,15 @@ void RelayScreen::onDraw(sf::RenderTarget& window)
         bool near_friendly = false;
         foreach(SpaceObject, obj, space_object_list)
         {
-            if ((!P<SpaceShip>(obj) && !P<SpaceStation>(obj)) || !obj->isFriendly(my_spaceship))
```

Comment:
For readability and performance, it's better to get the P<ShipTemplateBasedObject> cast once and store it in a local variable.